entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
Is that a docker standard? (this `entrypoint.sh` file)

Reply:
yes https://docs.docker.com/engine/reference/builder/#entrypoint. Using this doing 2 things
reading env varaibles from env which starts with REACT_APP and injecting it run time to env.js file and adding a query parameter to env.js so that browser wont cache this file. Helps in adding runtime variables to js app